App/qml/menus/Main.qml

```diff
@@ -143,6 +151,17 @@ Item {
                         onClicked: App.language = "da"
                     }
                 }
+                
+                Image {
+                    height: 20
+                    fillMode: Image.PreserveAspectFit
+                    source: App.getAsset('flags/de.png')
```

Comment:
Where's the actual image?
Images need to go in `DeadAscend/App/assets/flags/` - if possible please include an SVG version also in case we need to change the scale.